test.js

```diff
@@ -435,7 +435,14 @@ test('unist-util-visit', function (t) {
 
   t.test('should visit added nodes', function (st) {
     var tree = remark().parse('Some _emphasis_, **importance**, and `code`.')
-    var other = remark().use(gfm).parse('Another ~~sentence~~.').children[0]
+
+    // Unified doesn't (yet) let us specify parse result type, so all
+    // we know is that it's a node, but we know it is a parent, so we
```

Comment:
Right, to specific a result type, unified would need to know the contents of the markdown ahead of time.
And would potentially need a second implementation of the parser inside the typings to solve what node type would be returned.

```suggestion
    // Unified doesn't know parse result type,
    // all we know is that it's a node, but we know it is a parent, so we
```